src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -64,40 +63,57 @@ public void TestRootGetProcessById()
             Assert.Equal(1, p.Id);
         }
 
-        [Theory, InlineData(false), InlineData(true)] // Expected behavior varies on Windows and Unix. Refer to #23969
-        public void ProcessStart_TryOpenFolder_ThrowsWin32Exception(bool useShellExecute)
-        {
-            Win32Exception e = Assert.Throws<Win32Exception>(() => Process.Start(new ProcessStartInfo { UseShellExecute = useShellExecute, FileName = Path.GetTempPath() }));
-        }
-
-        [Fact, PlatformSpecific(TestPlatforms.Linux)]
+        [Theory, InlineData(true), InlineData(false)] // Expected behavior varies on Windows and Unix. Refer to #23969
         [OuterLoop("Opens program")]
-        public void ProcessStart_UseShellExecuteTrue_OpenFile_ThrowsIfNoDefaultProgramInstalledSucceedsOtherwise()
+        public void ProcessStart_UseShellExecute_OnUnix_ThrowsIfNoProgramInstalled(bool isFolder)
```

Comment:
This makes more sense as separate tests for Windows and Unix.

Reply:
Using `[PlatformSpecific(TestPlatforms.Windows)]` (or .OSX, .Linux, .AnyUnix)

Reply:
Probably you could use `PlatformSpecific.AnyUnix` and runtime check for the OSX / Linux differences.

Reply:
Disregard - missed this is the Unix file